php/BUILD.bazel

```diff
@@ -137,3 +139,48 @@ pkg_files(
     strip_prefix = strip_prefix.from_pkg(""),
     visibility = ["//pkg:__pkg__"],
 )
+
+################################################################################
+# PECL .tgz Release
```

Comment:
Do we want to separate this into php/ext/google/protobuf or php/ext? Right now, I have the BUILD rule for the package.xml generation stuff and templates in php/ext/google/protobuf.

Reply:
I'm fine depending on your rules in php/ext/google/protobuf but I want the pkg_tar/release rule to be in this file.